utils/polus-bfio-util/bfio/bfio.py

```diff
@@ -974,7 +974,7 @@ def __init__(self, file_path, image=None,
             self._metadata.image(0).Name = file_path
             self._metadata.image().Pixels.channel_count = self._xyzct['C']
             self._metadata.image().Pixels.DimensionOrder = bioformats.omexml.DO_XYZCT
-        elif image != None:
+        elif image is not None:
```

Comment:
Usually this should be included in a different PR, but I don't think it matters.

Reply:
Not sure how this got in. It must have been living in my repo and accidentally got merged back into my working branch.